src/content/docs/2/guide/plugins/index.mdx

```diff
@@ -2,12 +2,301 @@
 title: Plugins
 ---
 
-import Stub from '@components/Stub.astro';
+import CommandTabs from '@components/CommandTabs.astro';
 
-<Stub>
+{/* TODO: Link to windowing system, commands for sending messages, and event system */}
 
-- What is a Plugin?
-- [Developing a Plugin](/2/guide/plugins/develop-plugin)
-- Guide/Recipe for each plugin in https://github.com/tauri-apps/plugins-workspace/tree/v2
+Tauri offers a windowing system with web view functionality, a way to send messages between the Rust process and the web view, and an event system along with several tooling to enhance the development experience. By design, the Tauri core does not contain features not needed by everyone. Instead it offers a mechanism to add external functionalities into a Tauri application called plugins. Plugins are able to hook into the Tauri lifecycle, expose Rust code that relies on the web view APIs, handle commands with Rust, Kotlin or Swift code, and much more.
 
-</Stub>
+## Available Plugins
+
+The official Tauri plugins can be found on the [features and recipes page](/2/guide/list/). Recipes explaining how to combine plugins for common use cases as well as community plugins can also be found on that page.
+
+## Add Plugin to Project
+
+To add a plugin to a Tauri app, you can use the [`plugin add`](/2/reference/cli/#plugin-add) command.
+For example, to install the [notification plugin](/2/guide/notification) run:
+
+{/* TODO: `plugin add` is going to be renamed to `add` */}
```

Comment:
Is there an issue on GH for this we can include inline?

Reply:
I don't think so, I'll just implement this immediately.

Reply:
Thanks! I'll put that in the meta issue to at least track it so we won't forget about it (hopefully 😅 )

Reply:
```suggestion
{/* TODO: `plugin add` is going to be renamed to `add`: https://github.com/tauri-apps/tauri/pull/7744 */}
```